components/automate-ui/src/app/entities/projects/project.reducer.ts

```diff
@@ -65,7 +65,7 @@ export function projectEntityReducer(
 
   switch (action.type) {
     case ProjectActionTypes.GET_ALL:
-      return set(GET_ALL_STATUS, EntityStatus.loading, projectEntityAdapter.removeAll(state));
+      return set(GET_ALL_STATUS, EntityStatus.loading, state);
```

Comment:
As mentioned in the preamble, this one in particular causes the problem.
We start a `GET_ALL`, here we erase the list--and that manifests in the UI.
Sometime later--much later at slow network speed--the list gets restored via `GET_ALL_SUCCESS`.

We still need the `removeAll` for single `GET` calls I believe, but it does nothing for `GET_ALL` calls and, in fact, causes the problem addressed by this PR, so let's remove it.

Reply:
If i'm understanding this correctly, what we're essentially doing now is _replacing_ the state instead of clearing it out completely and then writing in new state.... so on slow connections, the users will see the old state until it is replaced?

Reply:
Exactly; which is at it should be. In many cases, the data would not have changed so they will only see a spinner appear, then disappear, with the underlying table unchanged.